samples/sample_multi_transcode/src/transcode_utils.cpp

```diff
@@ -238,6 +238,13 @@ void TranscodingSample::PrintHelp()
     msdk_printf(MSDK_STRING("                May be 1 in the case when -mss option is specified \n"));
     msdk_printf(MSDK_STRING("  -la_ext       Use external LA plugin (compatible with h264 & hevc encoders)\n"));
     msdk_printf(MSDK_STRING("  -vbr          Variable bitrate control\n"));
+    msdk_printf(MSDK_STRING("  -avbr         Average variable bitrate control\n"));
+    msdk_printf(MSDK_STRING("  -convergence  bitrate convergence period for avbr, in the unit of frame\n"));
+    msdk_printf(MSDK_STRING("  -accuracy     bitrate accuracy for avbr, in the range of [1, 100]\n"));
+    msdk_printf(MSDK_STRING("  -qvbr <quality>\n"));
+    msdk_printf(MSDK_STRING("                Quality-defined variable bitrate control\n"));
```

Comment:
Please provide the range here and below if possible.

Reply:
Done